src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -64,6 +64,13 @@ class Request
     public const METHOD_CONNECT = 'CONNECT';
     public const METHOD_QUERY = 'QUERY';
 
+    /**
+     * The HTTP methods that can be overridden.
+     *
+     * @var string[]|null
+     */
+    public static ?array $allowedHttpMethodOverride = null;
```

Comment:
Why not using a method?

Reply:
We could, but I thought this wouldn't provide much...

Reply:
Consistency maybe (and I know that the Request class is not super consistent here)?

Reply:
PR updated, with a getter+setter